pkg/output/output.go

```diff
@@ -172,6 +172,11 @@ func (w *StandardWriter) formatStandard(output *clients.Response) ([]byte, error
 		builder.WriteString(w.aurora.Yellow("self-signed").String())
 		builder.WriteString("]")
 	}
+	if w.options.MisMatched && mistmatchedCert(output.Host, append(cert.SubjectAN, cert.SubjectCN)) {
```

Comment:
Mismatched should be a field in the clients.ResponseCertificate structure and should be populated in pkg/clients instead of in the output layer as done now